llvm/docs/CommandGuide/llvm-symbolizer.rst

```diff
@@ -216,6 +224,12 @@ OPTIONS
   This can be used to perform lookups as if the object were relocated by the
   offset.
 
+.. option:: --approximate-line
+
+  Print the approximate non-zero line number that is nearest to before an
+  input address. Nearest lookup is performed by querying the line-table
+  structure for an address having non-zero line information in close proximity.
+
```

Comment:
I think that this can be simplified. I suggest:

> If an address does not have an associated line number, use the last
> line number from the current sequence in the line-table. Such lines
> are labeled as "approximate" in the output as they may be misleading.

Reply:
Done